resumeINDIGO.html

```diff
@@ -0,0 +1,132 @@
+<!Doctype html>
+<html>
+<head>
+	<link href = "style.css" type = "text/css" rel="StyleSheet">
+</head>
+<body>
+	<div class = "Container">
+
+		<div class="external1">
+       	 <div class="internal1">
+        	  <p class="a">Rajpreet Singh Nayyar</p>
+        	    <p class="b">Software Developer</p>
+        	    <br>
+        </div>
+    </div>
+<br>     
+		<div class = "Partition">
+		<span class = "SubPartitionLargeForText" id="LeftHeading">
+			<h3>PROFILE</h3>
+			<p class = "EduList">Seeking the position of software developer in a challenging environment where my uncommon, creativity, skills, knowledge and passion will be employed in giving clients the best output. </p>
+		</span>
+
+		<span class = "SubPartitionSmall"></span>
+		<span class = "SubPartitionLargeForText">
+			<img src = "raj.png" alt="rajpreet" class = "ProfileImage">	
+		</span>
+		<span class = "SubPartitionSmall"></span>
```

Comment:
why this span

Reply:
I've divided the <div> into 5 parts using 5 spans. The span with SubPartitionSmall class has a width set in the css file. I've used it to keep the symmetry between 5 spans. Removing it would result in an antiSymmetric div.